v1/transform-global.yaml

```diff
@@ -147,6 +147,49 @@ paths:
         - get_from_cx:
             request:
               uri: '{{options.cx_host}}/v1/list/pair/{from}/{to}'
+  /list/{tool}/{from}/{to}:
```

Comment:
Based on the end point's description, this is supposed to do the same as `/list/tool/{tool}`, but `from` and `to` and required here, so how can this end point list "all the language pairs for a given tool" ? What is this end point supposed to do exactly that is different from the one below?

Even if there is a legitimate need for this end point, we need to find a different path for it, because it conflicts with `/list/pair/` and `/list/tool/`

Reply:
Updating /list/tool/{tool} with from and to should be fine. Currently, /list/tool/{tool} gives list of tools available for all pairs, while /list/tool/{tool}/{from}/{to} will be useful to query particular tool. This is used in MT card in CX. @santhoshtr @Nikerabbit Please see this.

Reply:
The existing `/list/tool/{tool}` end point could then be expanded to allow for optional `from` and `to` parametres, so that if they are not given, then list all pairs, if only `from` is given list the pairs for the given source language and if both are given, then basically return just them if applicable. Would such a `/list/tool/{tool}{/from}{/to}` end point cover your use cases?